crm_phonecall/__init__.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import models
+from . import wizard
+from . import report
+
+
+xmlid_renames = [
```

Comment:
These hooks should be removed for version 10, as it only applies to version 9 (although they don't fail as there's an if condition first).